src/RazorSdk/Tool/Program.cs

```diff
@@ -49,5 +60,20 @@ public static int Main(string[] args)
 
             return result;
         }
+
+        private static void RegisterAssemblyResolutionEvents()
+        {
+            var roslynPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "..", "..", "..", "Roslyn", "bincore");
```

Comment:
`AppContext.BaseDirectory` instead of `AppDomain.CurrentDomain.BaseDirectory`

Reply:
Is there a particular reason to prefer one over the other?

Reply:
AppDomain is considered the legacy feature and is mostly shimmed in .NET Core. AppContext is the preferred replacement.